### ISSUE TITLE
13 December homepage programming update 

### ISSUE THREAD

**Issue by devaneymoz** [mozilla/addons-frontend#7088]
For the Thursday, **13 December** push, please update homepage content per details [here](https://docs.google.com/document/d/1g4RuGKW__eTHNd277rKPMM46ygonXzW68jSs2OXEkPM/edit#). 

@muffinresearch

**Comment by ioanarusiczki** [comment]
The shelves look good on AMO -dev. Since the guidelines are present instead of the HERO/CAROUSEL on AMO dev  they will be verified later this week on AMO stage.

**Comment by ioanarusiczki** [comment]
Since the new guide lines are going live on prod this week, the Homehero banners will no longer be displayed. 
I verified the shelves on AMO stage - FF63(Win10) using [ the document ](https://docs.google.com/document/d/1g4RuGKW__eTHNd277rKPMM46ygonXzW68jSs2OXEkPM/edit#heading=h.nz7t14frvpfb)
I created the weather reports and news boosters (as a checking creating the Mozilla collections with admin tools is working).
Final validation will be made on AMO prod.
